SCClassLibrary/Common/Control/Server.sc

```diff
@@ -410,6 +421,65 @@ Server {
 		};
 	}
 
+	prHandleClientLoginInfoFromServer { |newClientID, newMaxLogins|
```

Comment:
Could you perhaps move the private implementations further down? It is better to have the public interface coming first. E.g. here: https://github.com/adcxyz/supercollider/blob/8be07637fb9255c52c3a908a14d0b46b6f140e80/SCClassLibrary/Common/Control/Server.sc#L582